test/scripts/e2e_subs/e2e-logs.sh

```diff
@@ -17,6 +17,11 @@ function app_txid {
     grep -o -E 'txid [A-Z0-9]{52}' | cut -c 6- | head -1
 }
 
+function commit_round {
+    # Get the round that the app transaction was confirmed
+    grep -o -E 'committed in round [0-9]+' | rev | cut -d ' ' -f 1 | rev
```

Comment:
```suggestion
    grep -o -E 'committed in round [0-9]+' | cut -d ' ' -f 4
```
ignorable nit...

Reply:
Changed in [b38c1b9](https://github.com/algorand/go-algorand/pull/5611/commits/b38c1b99e1c45ab3eb267c0d917d0ab0010b1da5)